web/docs/about-vast/why-vast.md

```diff
@@ -0,0 +1,205 @@
+emerging use cases, such as data science and detection engineering, require
+additional data-centric workbenches.
+
+VAST *complements* a [SIEM][siem] nicely with the following use cases:
+
+- **Offloading**: route the high-volume telemetry to VAST that would otherwise
+  overload your SIEM or be cost-prohibitive to ingest. By keeping the bulk of
+  the data in VAST, you remove bottlenecks and can selectively forward the
+  activity that matters to your SIEM.
+
+- **Compliance**: VAST supports fine-grained retention configuration to meet
+  [GDPR](https://en.wikipedia.org/wiki/General_Data_Protection_Regulation) and
+  other regulatory requirements. When storage capacity needs careful management,
+  VAST's *compaction* feature allows for weighted ageing of your data, so that
+  you can specify relative importance of event types. Powerful *transforms*
+  allow you to anonymize, pseudonymize, or encrypt specific fields—either to
+  sanitize [PII data](https://en.wikipedia.org/wiki/Personal_data) on import, or
+  ad-hoc on export when data leaves VAST.
+
+- **Data Science**: The majority of SIEMs provide an API-only, low-bandwidth
+  access path to your security data. VAST is an [Arrow][arrow]-native engine
+  that offers unfettered high-bandwidth access so that you can bring your own
+  workloads, with your own tools, e.g., to run iterative clustering algorithms
+  or complex feature extraction in conjunction with machine learning.
+
+[siem]: https://en.wikipedia.org/wiki/Security_information_and_event_management
+[arrow]: https://arrow.apache.org
+
+:::note Recommendation
+Unlike a heavy-weight legacy SIEM, VAST is highly embeddable so that you can
+run it everywhere: containerized in the public cloud, on bare-metal appliances
+deep in the network, or at the edge.
+:::
+
+## VAST vs. Data Warehouses
+
+Data warehouses and
+[OLAP](https://en.wikipedia.org/wiki/Online_analytical_processing) engines
+seem like an appealing choice for immutable structured data. They offer
+sufficient ingest bandwidth, perform well on group-by and aggregation queries,
+come frequently with advanced operations like joins, and often scale out well.
+
+However, as a cornerstone for security operations, they fall short in supporting
+the following relevant use cases where VAST has the edge:
+
+- **Data Onboarding**: it takes considerable effort to write and maintain
+  schemas for the tables of the respective data sources. Since VAST is
+  purpose-built for security data, integrations for key data sources and data
+  carriers exist out of the box.
+
+- **Rich Typing**: modeling security event data with a generic database often
+  reduces the values to strings or integers, as opposed to retaining
+  domain-specific semantics, such as IP addresses or port numbers. VAST offers a
+  rich type system that can retain such semantics, supporting both
+  *schema-on-read* (taxonomies) and *schema-on-write* (transforms).
```

Comment:
I removed it here to avoid confusion. The idea is that schema-on-write fixates the schema on ingest. I realized it's actually bad use of terminology, because schema-on-read typically means "parse the raw blob later at query time", which we are not doing.